### DISCUSSION TITLE
VRP with multiple depots and route durations

### DISCUSSION THREAD

**Discussion by abhaysobhanan** [google/or-tools#3218] [Routing (and legacy CP) questions] [answered]
Does the current version support solving MDVRP instances with route duration?

**Comment by Mizux** [comment, marked as answer]
vrp: yes
multiple depots: yes
time dimension i.e. vehicle max duration: yes
time windows: yes

**Reply by abhaysobhanan** [reply]
Is there any example or test cases for handling multiple depots? I could not find one in the given list of examples.

**Reply by Mizux** [reply]
https://github.com/google/or-tools/blob/stable/ortools/constraint_solver/samples/vrp_starts_ends.py ?
